app/assets/stylesheets/common.scss

```diff
@@ -1086,4 +1086,102 @@ div.secondary-actions {
   }
 }
 
+.auth-container {
```

Comment:
I haven't had the chance for a full review, but these stylesheet changes stand out to me. Our general approach is to use the bootstrap framework, not custom CSS. In particular, custom max-widths, custom font sizes, custom border radii etc aren't the way we should be implementing changes.

Instead, bootstrap classes should be added to html elements, e.g. `rounded-2` should be used. These can of course be used sparingly, since html elements like `<small>` should do the right thing too.

I can see an implementation of tabs here too, and I would expect instead to see bootstrap tabs being used, as elsewhere on the site.

Reply:
I updated the PR to use minimal set of custom stylesheets, as suggested.

The only remaining pieces are the total width of the screen in signup and login screens and customization of bootstrap tabs for line style. Granted, the tab style is not same as currently used in OSM in GPX traces, it wouldn't work without darker background.